test_wakeonlan.py

```diff
@@ -78,6 +78,27 @@
             b"\xff\xff\xff\xff\xff\xff"
             b"\xff\xff\xff\xff\xff\xff",
         ),
+        (
+            "ffff.ffff.ffff",
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff"
+            b"\xff\xff\xff\xff\xff\xff",
+        ),
+
     ],
     ids=["no separator", "colons", "hyphens"],
```

Comment:
Pytest won't run without an id for this test.

```suggestion
        ),
    ],
    ids=["no separator", "colons", "hyphens", "dots"],
```